lib/ethereumex/http_client.ex

```diff
@@ -31,6 +31,12 @@ defmodule Ethereumex.HttpClient do
     end
   end
 
+  defp headers(opts) do
+    headers = Keyword.get(opts, :http_headers) || Config.http_headers()
+
+    headers ++ [{"Content-Type", "application/json"}]
```

Comment:
Is there a reason to prefer append to prepend here? Maybe something like.

```suggestion
    [{"Content-Type", "application/json"} | headers]
```